cypress/integration/backoffice/overview.test.spec.js

```diff
@@ -0,0 +1,30 @@
+describe('backoffice overview', () => {
+  const login = () => {
+    cy.visit('/login');
+    cy.get('input#email').type('librarian@test.ch');
+    cy.get('input#password').type('123456');
+    cy.contains('button.primary', 'Sign in').click();
+  };
+
+  it('should be linked from/to the frontsite, can logout', () => {
+    login();
+
+    cy.contains('Hector Nabu').click();
+    cy.contains('Backoffice').click();
+
+    cy.url().should('eq', Cypress.config().baseUrl + '/backoffice');
+    cy.contains('.active', 'Overview');
+
+    cy.contains('Go to Frontsite').click();
+
+    cy.url().should('eq', Cypress.config().baseUrl + '/');
+
+    cy.go('back');
+
+    cy.url().should('eq', Cypress.config().baseUrl + '/backoffice');
+
+    cy.contains('Sign out').click();
```

Comment:
disclaimer: I haven't use Cypress before

Proposal for idea, I see we are using the text of the html elements to refer to them. Would it make sense to create a shared structure like 
```
{
  pages: {
    login: {
      button: 'Sign In'
    }
  }
}
```
we will achieve reusability in our cases but most importantly if we change an element we will have to change it in one place only. Just a proposal.

Reply:
That also a question I asked to myself, how to factor common logic in this framework. For instance a lot of the tests will require the user to be logged in; instead of signing in the user each time using the UI we could take a shortcut and perform a request to the backend instead.
I also saw a mention of hooks that are run before/after a test, this should be looked into details as it will help us share common actions / aliases.